osu.Game/Skinning/SkinInfo.cs

```diff
@@ -39,6 +39,21 @@ public class SkinInfo : RealmObject, IHasRealmFiles, IEquatable<SkinInfo>, IHasG
 
         public bool Protected { get; set; }
 
+        /// <summary>
+        /// The latest version in YYYYMMDD format for skin layout migrations.
+        /// </summary>
+        /// <remarks>
+        /// <list type="bullet">
+        /// <item><description>20240625: Moves combo counters from ruleset-agnostic to ruleset-specific HUD targets.</description></item>
+        /// </list>
+        /// </remarks>
+        public const int LATEST_LAYOUT_VERSION = 20240625;
+
+        /// <summary>
+        /// A version in YYYYMMDD format for applying skin layout migrations.
+        /// </summary>
+        public int LayoutVersion { get; set; }
```

Comment:
Why isn't this set to `LATEST_LAYOUT_VERSION` here? Why is it set in the importer?

Compare e.g.

https://github.com/ppy/osu/blob/76a1f19233105a22c11803010453f85cfdf4ab3f/osu.Game/Scoring/ScoreInfo.cs#L91

Reply:
Makes sense, addressed in https://github.com/ppy/osu/pull/26249/commits/0c34e7bebbce5a989af00d8daed25331ff33321a.